composer.json

```diff
@@ -26,7 +30,7 @@
         "gedmo/doctrine-extensions": "^3.0",
         "matthiasnoback/symfony-dependency-injection-test": "^4.2.1",
         "symfony/cache": "^4.4 || ^5.2",
-        "symfony/framework-bundle": "^4.4 || ^5.2",
```

Comment:
Why this removal ?

Reply:
Class `Controller` was removed in `symfony/framework-bundle:5.x`:
https://github.com/sonata-project/EntityAuditBundle/blob/af61a7bfcec2052c381b5c3333a0ccbe6d7d7b3b/src/Controller/AuditController.php#L23-L37